.github/workflows/android.yml

```diff
@@ -0,0 +1,98 @@
+name: android
+
+on: [push]
+
+jobs:
+  linuxdist:
+    name: linux_dist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - name: 'Install dependencies'
+        run: ./ci/linux_ci_setup.sh
+      - name: 'Build envoy.aar distributable'
+        run: |
+          export PATH=/usr/lib/llvm-8/bin:$PATH
+          export CC=clang
+          export CXX=clang++
+          export ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk-bundle
+          ./bazelw build --fat_apk_cpu=x86 //:android_dist
+    # TODO: parallelize these two jobs
+      - name: 'Build Java app'
+        run: |
+          export ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk-bundle
+          ./bazelw build --fat_apk_cpu=x86 //examples/java/hello_world:hello_envoy
+      - name: 'Build kotlin app'
+        run: |
+          export ANDROID_NDK_HOME=/usr/local/lib/android/sdk/ndk-bundle
+          ./bazelw build --fat_apk_cpu=x86 //examples/kotlin/hello_world:hello_envoy_kt
+  macdist:
+    name: mac_dist
+    runs-on: macos-latest
```

Comment:
Are we sure that running on the latest here is always safe?

Reply:
probably not. I can pin to a version in a subsequent PR.

Reply:
sounds good!